lib/gis/parser_json.c

```diff
@@ -340,19 +341,30 @@ char *G__json(void)
  */
 void check_create_import_opts(struct Option *opt, char *element, FILE *fp)
 {
-    int i = 0;
+    int i = 0, urlfound;
     int has_import = 0;
     char **tokens;
 
+    G_debug(2,"tokenize opt string: <%s> with '@'", opt->answer);
     tokens = G_tokenize(opt->answer, "@");
     while (tokens[i]) {
         G_chop(tokens[i]);
         i++;
     }
 
+    /* check if tokens[1] starts with an URL or name@mapset */
```

Comment:
This block needs to be enclosed in a `if (i > 1) {`, i.e. do not investigate token[1] if there is only one token.
Fatal error if i > 2?

Reply:
OK, updated in 7d0d68ecb